backend/src/common/enums/app/env.enum.ts

```diff
@@ -28,6 +32,12 @@ const ENV = {
   API: {
     V1_PREFIX: '/api/v1',
   },
+  MAILTRAP: {
+    EMAIL_HOST: EMAIL_HOST,
+    EMAIL_USERNAME: EMAIL_USERNAME,
+    EMAIL_PASSWORD: EMAIL_PASSWORD,
+    FROM_EMAIL: FROM_EMAIL,
+  },
```

Comment:
Shouldn't all these vars be under the `NODEMAILER` object? Because, as I understand, we use the `nodemailer` instead of the `mailtrap`. Which is totally cool, but I just want to be consistent :slightly_smiling_face: